docker-compose/docker-compose.override.yml

```diff
@@ -4,32 +4,46 @@ services:
   armonik.control.submitter:
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
+      - Components__QueueStorage=ArmoniK.Adapters.Amqp.QueueStorage
 
   armonik.control.metrics:
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
 
   armonik.compute.pollingagent0:
+    depends_on:
+      queue:
+        condition: service_started
```

Comment:
Why in the override and not the regular docker-compose.yml ?
A bit of explanation for this should be better. Does it uses our health checks ?

Reply:
It is already in the regular compose file. I forgot to remove it from here ^^. I'll add some doc.